[2023-12-20 16:14:07] {UserService.py:530} ERROR - Error invalidating reset token: (psycopg2.errors.UndefinedTable) relation "bookmazon.password_reset_tokens" does not exist
LINE 1: UPDATE bookmazon.password_reset_tokens SET token_status='2',...
               ^

[SQL: UPDATE bookmazon.password_reset_tokens SET token_status=%(token_status)s, update_datetime=%(update_datetime)s WHERE bookmazon.password_reset_tokens.token = %(token_1)s]
[parameters: {'token_status': '2', 'update_datetime': datetime.datetime(2023, 12, 20, 16, 14, 7, 337601), 'token_1': 'dJQTLDrKwKs3GQEUjKzUeLVge1dw_aKRz7DpVk2raLs'}]
(Background on this error at: https://sqlalche.me/e/20/f405)
[2023-12-20 16:14:07] {UserService.py:530} ERROR - Error invalidating reset token: (psycopg2.errors.UndefinedTable) relation "bookmazon.password_reset_tokens" does not exist
LINE 1: UPDATE bookmazon.password_reset_tokens SET token_status='2',...
               ^

[SQL: UPDATE bookmazon.password_reset_tokens SET token_status=%(token_status)s, update_datetime=%(update_datetime)s WHERE bookmazon.password_reset_tokens.token = %(token_1)s]
[parameters: {'token_status': '2', 'update_datetime': datetime.datetime(2023, 12, 20, 16, 14, 7, 337601), 'token_1': 'dJQTLDrKwKs3GQEUjKzUeLVge1dw_aKRz7DpVk2raLs'}]
(Background on this error at: https://sqlalche.me/e/20/f405)
[2023-12-20 16:14:07] {UserService.py:530} ERROR - Error invalidating reset token: (psycopg2.errors.UndefinedTable) relation "bookmazon.password_reset_tokens" does not exist
LINE 1: UPDATE bookmazon.password_reset_tokens SET token_status='2',...
               ^

[SQL: UPDATE bookmazon.password_reset_tokens SET token_status=%(token_status)s, update_datetime=%(update_datetime)s WHERE bookmazon.password_reset_tokens.token = %(token_1)s]
[parameters: {'token_status': '2', 'update_datetime': datetime.datetime(2023, 12, 20, 16, 14, 7, 337601), 'token_1': 'dJQTLDrKwKs3GQEUjKzUeLVge1dw_aKRz7DpVk2raLs'}]
(Background on this error at: https://sqlalche.me/e/20/f405)
[2023-12-20 16:14:07] {UserService.py:530} ERROR - Error invalidating reset token: (psycopg2.errors.UndefinedTable) relation "bookmazon.password_reset_tokens" does not exist
LINE 1: UPDATE bookmazon.password_reset_tokens SET token_status='2',...
               ^

[SQL: UPDATE bookmazon.password_reset_tokens SET token_status=%(token_status)s, update_datetime=%(update_datetime)s WHERE bookmazon.password_reset_tokens.token = %(token_1)s]
[parameters: {'token_status': '2', 'update_datetime': datetime.datetime(2023, 12, 20, 16, 14, 7, 337601), 'token_1': 'dJQTLDrKwKs3GQEUjKzUeLVge1dw_aKRz7DpVk2raLs'}]
(Background on this error at: https://sqlalche.me/e/20/f405)
